content/4_Gold/Intro_DP.mdx

```diff
@@ -128,16 +125,15 @@ problemsetters rarely set direct applications of them.
 		url="https://atcoder.jp/contests/dp/tasks"
 		starred
 	>
-		Some tasks are beyond the scope of Gold.
+		Some tasks are beyond the scope of Gold. Editorials for problems A to I can be found [here](https://nwatx.me/post/atcoderdp). Alternative implementations for problems A to L can be found [here](https://github.com/jessechoe10/CP/tree/master/AtCoder/DP%20Contest).
```

Comment:
@jessechoe10 doesn't contain J

Reply:
sheesh i can add later.

Reply:
I'll add more solutions later lol (it shouldn't be too hard to add them)

Reply:
PRs should reflect the current state 👀

Reply:
What should I change it to then?

Reply:
nwatx has A-O

Reply:
sussy irl